engines/agi/agi.h

```diff
@@ -232,6 +232,11 @@ struct gameIdList {
 struct Mouse {
 	int button;
 	Common::Point pos;
+
+	void reset() {
+		button = 0;
+		pos.x = pos.y = 0;
```

Comment:
Wouldn't it be better to use the default ctor here? What if the class gets more parameters in the future. You might end up with garbage values in that case.

Reply:
I've changed the initialization of Mouse and AgiGame to what I _think_ you meant. I did revert some such changes I had made before even committing them, but I missed those.